libraries/abstractions/common_io/test/test_iot_gpio.c

```diff
@@ -148,29 +122,17 @@ TEST_GROUP_RUNNER( TEST_IOT_GPIO )
     TEST_ASSERT_NOT_EQUAL( NULL, xtestIotGpioSemaphore );
 
     RUN_TEST_CASE( TEST_IOT_GPIO, AFQP_IotGpioOpenClose );
-    testIotGpioClosePorts();
```

Comment:
reason we had this originally is to make sure ports are closed if previous test exit early because of assertion and not closing port. if we remove this, how do we make sure early test will not affect later ones?

Reply:
ok, I see each test case has setup and teardown, and we close port in tear down phase, is this true for every RUN_TEST_CASE? changes look ok if that's the case

Reply:
Yes, that is true. That is how setup/teardown are supposed to be used.